### DISCUSSION TITLE
Osu!lazer wont open on MacOS anymore

### DISCUSSION THREAD

**Discussion by BeetrootJuice** [ppy/osu#23035] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

Hi.

Yesterday i opened osu!lazer as usuall, and then information about update poped out so i clicked it. New version of osu downloaded, i uninstalled the previous one (i dont know why it works like it's downloading osu one more time). And from that point Osu! dont want to even open. Every time i try its not even turning on. I tried reinstalling three times, but the situation doesnt change. I'm on MacOS BigSur 11.7.3. I'll be glad if you help me. Thanks in advance

### Version

I dont know exact number but its the version from 28 march 2023

### Logs

[database.log](https://github.com/ppy/osu/files/11085754/database.log)
[input.log](https://github.com/ppy/osu/files/11085755/input.log)
[network.log](https://github.com/ppy/osu/files/11085756/network.log)
[performance.log](https://github.com/ppy/osu/files/11085757/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11085758/runtime.log)

**Comment by peppy** [comment]
Is there a reason you are on such an old OS version?

**Reply by peppy** [reply]
This is likely a duplicate of https://github.com/ppy/osu/issues/23021, similar old version.

**Comment by BeetrootJuice** [comment]
I lack of memmory to install new versions and I have no time to hard reset the computer, but I will do it eventually :)

**Comment by Joehuu** [comment, marked as answer]
Should be fixed in the latest release.
